bindings/generator/generate.py

```diff
@@ -446,6 +488,13 @@ class ParsingPlaceholder:
             placeholder.__dict__ = struct.__dict__
             placeholder.__class__ = struct.__class__
             structs.append(placeholder)  # type: ignore[arg-type]
+        elif isclass(item) and issubclass(item, Enum):
+            placeholder = TYPES_DB[item]
+            annotations = getattr(item, "__annotations__", {})
+            enum = EnumInUse(member_names=item._member_names_, name=item.__name__)
+            placeholder.__dict__ = enum.__dict__
+            placeholder.__class__ = enum.__class__
```

Comment:
`TYPES_DB`'s type should be updated to avoid lint error here

Reply:
I don't understand, other types don't update `TYPES_DB` (struct, variant).
Can you provide an snippet of what is expected ?

Reply:
I was refering to `TYPES_DB: dict[type, OpaqueSpec | StructSpec | VariantSpec | EnumSpec]` which was missing `EnumSpec`, this is all good now ;-)